### PR TITLE
BigInt docs

### PR DESCRIPTION
**Pull request by fhammerschmidt**

### CODE REVIEW

File: pages/docs/manual/latest/primitive-types.mdx [hunk 1]
```diff
@@ -166,6 +166,95 @@ var result = 1 + 2;
...
+```
+```js
+var a = 9007199254740991n + 9007199254740991n;
+
+var p = 2n ** 2n;
+```
+
+</CodeTab>
+
+It also supports all the bitwise operations.
```
Comment (mununki) [inline pages/docs/manual/latest/primitive-types.mdx:196]: Can we mention that supporting all the bitwise operations except `lsr`? which is not supported in Js built-in apis.

Comment (fhammerschmidt) [reply pages/docs/manual/latest/primitive-types.mdx]: It's now mentioned!

File: pages/docs/manual/latest/primitive-types.mdx [hunk 2]
```diff
@@ -166,6 +166,95 @@ var result = 1 + 2;
...
+We provide the usual operations on them: `+`, `-`, `*`, `/`, etc. See [BigInt](api/core/bigint) for helper functions.
+
+A `bigint` number is denoted by a trailing `n` like so: `42n`.
+
+As `bigint` is a different data type than `int`, it's necessary to open the corresponding module to overload the operators.
+
+<CodeTab labels={["ReScript", "JS Output"]}>
+
+```res example
+open! Js.BigInt
```
Comment (fhammerschmidt) [inline pages/docs/manual/latest/primitive-types.mdx:183]: I'd prefer to just use `open! Bigint` when RescriptCore is opened, but that does not work yet, a new Core version is needed.

**Review by mununki — APPROVED** [review]
Great! thanks!

**Review by zth — APPROVED** [review]
Nice!

**Pull request merged**